src/storage/compressed_tile.cpp

```diff
@@ -0,0 +1,588 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// compressed_tile.cpp
+//
+// Identification: src/storage/compressed_tile.cpp
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include <cstdio>
+#include <sstream>
+
+#include "catalog/schema.h"
+#include "common/exception.h"
+#include "common/macros.h"
+#include "type/serializer.h"
+#include "type/types.h"
+#include "type/ephemeral_pool.h"
+#include "type/value_factory.h"
+#include "concurrency/transaction_manager_factory.h"
+#include "storage/storage_manager.h"
+#include "storage/tile.h"
+#include "storage/tile_group_header.h"
+#include "storage/tuple.h"
+#include "storage/tuple_iterator.h"
+#include "storage/compressed_tile.h"
+#include "container/cuckoo_map.h"
+
+namespace peloton {
+namespace storage {
+
+bool CompareLessThanBool(type::Value left, type::Value right) {
+  return left.CompareLessThan(right);
+}
+
+// Constructor: Call base constructor and set is_compressed to false.
+CompressedTile::CompressedTile(BackendType backend_type,
+                               TileGroupHeader *tile_header,
+                               const catalog::Schema &tuple_schema,
+                               TileGroup *tile_group, int tuple_count)
+    : Tile(backend_type, tile_header, tuple_schema, tile_group, tuple_count) {
+  is_compressed = false;  // tile is currently uncompressed.
+  compressed_columns_count = 0;
+}
+
+CompressedTile::~CompressedTile() {}
+
+// CompressTile
+// @brief: This function identifies the columns that can be compressed and
```

Comment:
Just one suggestion: Could you unify the comments format in this file? Sometimes you use `//@breif` and sometimes you use `/*@brief`.